gcp/appengine/frontend_handlers.py

```diff
@@ -242,6 +242,24 @@ def vulnerability_redirector(potential_vuln_id):
   return None
 
 
+@blueprint.route('/<potential_vuln_id>.json')
+@blueprint.route('/vulnerability/<potential_vuln_id>.json')
+def vulnerability_json_redirector(potential_vuln_id):
+  """Convenience redirector for /VULN-ID.json and /vulnerability/VULN-ID.json to
+  https://api.osv.dev/v1/vulns/VULN-ID.
+  """
+  if not _VALID_VULN_ID.match(potential_vuln_id):
+    abort(404)
+    return None
+
+  vuln = osv_get_by_id(potential_vuln_id)
```

Comment:
minor nit: can you change the `if vuln:` to `if not vuln:` to have the final return statement be the redirect, and the if branches be the aborts.